testgrid/cmd/configurator/main.go

```diff
@@ -273,7 +273,11 @@ func main() {
 	var client *storage.Client
 	if strings.HasPrefix(opt.output, "gs://") {
 		var err error
-		client, err = gcs.ClientWithCreds(ctx, opt.creds)
+		var creds []string
+		if opt.creds != "" {
+			creds = append(creds, opt.creds)
```

Comment:
It looks like you're appending opt.creds to `""`. What is this snippet doing that's different from how it was before?

Reply:
Your reading it backwards. If creds are non empty then add send it to ClientWithCreds, otherwise use do not send this argument (which causes us to use default credentials, what we want).